doc/source/tokudb/using_tokudb.rst

```diff
@@ -1,33 +1,303 @@
-.. _using_tokudb:
+.. _using-tokudb:
 
-==============
- Using TokuDB
-==============
+============
+Using TokuDB
```

Comment:
Several links below referring to tokutek.com which is flatly redirected to Percona.com, all Tokutek.com content is gone.